app/controllers/projects_controller.rb

```diff
@@ -1,17 +1,30 @@
 class ProjectsController < ApplicationController
-  before_action :set_project, except: %i[new create index archived]
+  before_action :set_project, except: %i[new create index archived join]
 
   Project::MAX_MEMBERS_PER_CARD = 4;
 
   # GET /projects
   # GET /projects.xml
   def index
-    @projects = policy_scope(Project).not_archived
-    @activities_group = Activity.grouped_activities(@projects, 1.week.ago)
+    @projects = Hash.new
+
+    current_user_projects = policy_scope(Project)
+
+    @projects[:is_member] = Projects::ProjectsSerializer::from_collection(
+      ProjectDecorator.decorate_collection(current_user_projects)
+    )
+
+    @projects[:without_current_user] = Projects::ProjectsSerializer::from_collection(
+      ProjectDecorator.decorate_collection(
+        current_team.projects.joinable.where.not(id: current_user.projects).order(:updated_at)
```

Comment:
PS: You're also using `current_team.projects` and `current_user.projects` in some places. If you like and implement my suggestion, you can just call `user_projects ` and `team_projects` instead.